src/main/c/cext/ruby.c

```diff
@@ -2799,7 +2799,10 @@ NORETURN(void rb_eof_error(void)) {
 }
 
 VALUE rb_io_addstr(VALUE io, VALUE str) {
-  return (VALUE) truffle_invoke(io, "<<", str);
+  // use write instead of just #<<, it's closer to what MRI does
```

Comment:
MRI's `rb_io_addstr` seems to call `rb_io_write` which calls `io.write(str)`, so is there any difference?

Reply:
reusing rb_io_write